src/OMSimulatorLib/OMSimulator.h

```diff
@@ -99,6 +99,7 @@ OMSAPI oms_status_enu_t OMSCALL oms_getInteger(const char* cref, int* value);
 OMSAPI oms_status_enu_t OMSCALL oms_getModelState(const char* cref, oms_modelState_enu_t* modelState);
 OMSAPI oms_status_enu_t OMSCALL oms_getReal(const char* cref, double* value);
 OMSAPI oms_status_enu_t OMSCALL oms_getResultFile(const char* cref, char** filename, int* bufferSize);
+OMSAPI oms_status_enu_t OMSCALL oms_getSignalFilter(const char* cref, char** regex);
```

Comment:
This need to be added to the documenatation.

Reply:
added the documentation